apps/bridge-dapp/src/containers/DepositContainer/DepositContainer.tsx

```diff
@@ -537,6 +491,236 @@ export const DepositContainer = forwardRef<
       setDestChain(defaultDestinationChain);
     }, [defaultDestinationChain]);
 
+    const tokenListDepositProps = useMemo<TokenListCardProps>(() => {
+      return {
+        className: 'w-[550px] h-[700px]',
+        title: 'Select Asset to Deposit',
+        popularTokens: [],
+        selectTokens: populatedSelectableWebbTokens,
+        unavailableTokens: populatedAllTokens,
+        onChange: handleTokenChange,
+        onClose: () => setMainComponentName(undefined),
+        onConnect: onTryAnotherWallet,
+      };
+    }, [
+      populatedSelectableWebbTokens,
+      populatedAllTokens,
+      handleTokenChange,
+      setMainComponentName,
+      onTryAnotherWallet,
+    ]);
+
+    const tokenListWrapAndDepositProps = useMemo<
+      TokenListCardProps | undefined
+    >(() => {
+      if (!wrappableCurrency || !bridgeGovernedCurrency) {
+        return undefined;
+      }
+
+      const tokens = getPossibleGovernedCurrencies(wrappableCurrency.id).map(
+        (currency): AssetType => ({
+          name: currency.view.name,
+          balance: balances[currency.id] ?? 0,
+
+          symbol: currency.view.symbol,
+        })
+      );
+
+      return {
+        className: 'w-[550px] h-[700px]',
+        overrideScrollAreaProps: { className: 'h-[550px]' },
+        chainType: 'dest',
+        selectTokens: tokens,
+        value: destChainInputValue,
+        title: 'Select Asset to Deposit',
+        popularTokens: [],
+        unavailableTokens: populatedAllTokens,
+        onChange: (selectedChain) => {
+          const destChain = Object.values(chains).find(
+            (val) => val.name === selectedChain.name
+          );
+          setDestChain(destChain);
+          setMainComponentName(undefined);
+        },
+      };
+    }, [
+      destChainInputValue,
+      destChains,
+      setDestChain,
+      setMainComponentName,
+      populatedAllTokens,
```

Comment:
`React Hook useMemo has missing dependencies: 'balances', 'bridgeGovernedCurrency', 'chains', 'getPossibleGovernedCurrencies', and 'wrappableCurrency'.`